charts/fleet/charts/gitjob/values.yaml

```diff
@@ -15,3 +15,12 @@ global:
 
 # comma separated list of domains or ip addresses that will not use the proxy
 noProxy: 127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16,.svc,.cluster.local
+
+nodeSelector:
```

Comment:
For gitjob you have to change the chart at https://github.com/rancher/gitjob.git and update it.

Reply:
Ah, that makes sense.  I was unsure if this chart was manually pulled in and modified. Will do.

Reply:
PR: https://github.com/rancher/gitjob/pull/29